images/python-predictor-gpu/Dockerfile

```diff
@@ -44,11 +33,50 @@ RUN /opt/conda/bin/conda create -n env -c conda-forge python=$PYTHONVERSION pip
 
 ENV BASH_ENV=~/.env
 
-COPY pkg/workloads/cortex/lib/requirements.txt /src/cortex/lib/requirements.txt
-COPY pkg/workloads/cortex/serve/python-gpu.requirements.txt /src/cortex/serve/requirements.txt
+COPY pkg/workloads/cortex/serve/requirements.txt /src/cortex/serve/requirements.txt
 
 RUN /bin/bash -c \
-    "pip install --no-cache-dir -r /src/cortex/lib/requirements.txt -r /src/cortex/serve/requirements.txt"
+    "pip install --no-cache-dir \
+        -r /src/cortex/serve/requirements.txt"
+
+ARG SLIM="false"
+RUN test "${SLIM}" != "true" && \
+        # libsm6, libxext6, and libxrender-dev are required for opencv
+        # libpng-dev is required for some image processing libaries
+        # libsndfile1 is required for some audio processing libaries
+        # libnvinfer is required for TensorRT
+        apt-get update -qq && apt-get install -y -q \
+            libsm6 \
+            libxext6 \
+            libxrender-dev \
+            libpng-dev \
+            libsndfile1 \
+            libnvinfer6=6.0.1-1+cuda10.1 \
+            libnvinfer-plugin6=6.0.1-1+cuda10.1 \
+        && apt-get clean -qq && rm -rf /var/lib/apt/lists/* \
+        && /bin/bash -c \
+            "pip install --no-cache-dir \
+                cloudpickle==1.3.0 \
+                Cython==0.29.16 \
+                joblib==0.14.1 \
+                Keras==2.3.1 \
+                nltk==3.4.5 \
+                np-utils==0.5.12.1 \
+                opencv-python==4.2.0.32 \
+                pandas==1.0.3 \
+                Pillow==7.0.0 \
+                scikit-image==0.16.2 \
+                scikit-learn==0.22.2.post1 \
+                scipy==1.4.1 \
+                six==1.14.0 \
+                statsmodels==0.11.1 \
+                sympy==1.5.1 \
+                tensorflow-gpu==2.1.0 \
+                tensorflow-hub==0.7.0 \
+                torch==1.4.0 \
+                torchvision==0.5.0 \
+                xgboost==1.0.2" \
+    || true
```

Comment:
If there is a failure in one of the pip installs or apt-get would this `|| true` swallow the failure and still move to the next line? I think something along the lines of `test "${SLIM}" == "true" || (apt-get .... )" might be safer. I am not sure what if shell boolean logic short circuits conditions

Reply:
Good catch!!!